app/components/content/accordion_component.rb

```diff
@@ -0,0 +1,35 @@
+module Content
+  class AccordionComponent < ViewComponent::Base
+    include ViewComponent::Slotable
```

Comment:
Nice, I've never seen this used before. Looks like its experimental still so one to keep an eye on when we upgrade the gem.